### PR TITLE
Documentation for the NADTelnet platform

### PR DESCRIPTION
**Pull request by andreas-amlabs**
**Description:**

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#11524

## Checklist:

  - [ ] Branch: Fixes, changes and adjustments should be created against `current`. New documentation for platforms/components and features should go to `next`. 
  - [ ] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

### CODE REVIEW

**Comment by andreas-amlabs** [comment]
Documentation matching
https://github.com/home-assistant/home-assistant/pull/11524

Which is not yet approved

File: source/_components/media_player.nadtelnet.markdown [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+To add an NAD receiver to your installation, add the following to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+media_player:
+  - platform: nadtelnet
+    host: 192.168.0.150
+```
+
+Configuration variables:
```
Comment (frenck) [inline source/_components/media_player.nadtelnet.markdown]: Please use the configuration tags, for more information please see:
https://home-assistant.io/developers/documentation/create_page/#configuration

**Review by frenck — CHANGES_REQUESTED** [review]
@andreas-amlabs Thank you for sending in the documentation, could you please take a look at my review comment? Thanks 👍

**Comment by andreas-amlabs** [comment]
Gave it a go, hope it went well

**Comment by andreas-amlabs** [comment]
A little help plz, what is wrong with

+sources:
 +  description: A list of mappings from sources to source name.
 +  required: false
 +  type: list
 +  keys:
 +    1:
 +      description: A valid source description

**Comment by andreas-amlabs** [comment]
So keep getting "Liquid Exception: undefined method `downcase' for ["1-10"]:Array" no matter what

Trying to ask for help again, please assist

**Comment by frenck** [comment]
This PR is still on my radar don't worry. I've assigned myself to look at it on a short term.

**Comment by frenck** [comment]
Closing this PR, since the parent PR has been closed as well.

**Pull request closed without merging**